curl-to-elisp.el

```diff
@@ -285,6 +291,29 @@ Return nil if S does not contain CH."
    command
    (rx bos (* blank) (? (in "$#")) (* blank))))
 
+
+;;; ###autoload
+(defun curl-to-elisp-httpie-to-elisp (command)
+  "Convert httpie/curlie COMMAND to Emacs Lisp expression, return the expression.
+
+When called interactively, also pretty-print the expression in echo area."
+  (interactive "shttpie command: ")
+  (if curl-to-elisp-curlie-binary
+      (let ((command (replace-regexp-in-string
+                      "^\\(curlie\\|http\\) "
```

Comment:
Why http (I assume it's httpie's executable) as well? does curlie support httpie's options?

Reply:
curlie copies the httpie interface, so it's kinda safe to accept both curlie and http prefixes.

Reply:
OK, make sense